ci-operator/config/openshift/assisted-service/openshift-assisted-service-master__edge.yaml

```diff
@@ -109,14 +109,18 @@ releases:
     release:
       channel: candidate
       version: "4.8"
-  initial:
+  four-nine:
```

Comment:
Can we change to release-4-10?

Reply:
We want this to be 4.9 so that it would independent of `initial` or `latest` as its used in
* e2e-ai-operator-ztp-ipv4v6-sno-ocp-49
* e2e-ai-operator-ztp-ipv4v6-sno-periodic-ocp-49
* e2e-ai-operator-ztp-ipv4v6-3masters-ocp-49
* e2e-ai-operator-ztp-ipv4v6-3masters-periodic-ocp-49

Reply:
sorry, meant ``release-4-9``
somehow my mistake proved my point, words cannot describe numbers in a good way :joy:

Reply:
It's kind of following our convention in some other places: https://github.com/openshift/release/blob/e799afc4f462f0f7da329fda40c2435248ef740f/ci-operator/config/openshift/assisted-service/openshift-assisted-service-release-ocm-2.5.yaml#L100

Reply:
https://github.com/openshift/release/pull/29463